Source/settings/maps/Route 21.js

```diff
@@ -11,6 +11,39 @@ FullScreenPokemon.FullScreenPokemon.settings.maps.library["Route 21"] = {
             "allowCycling": true,
             "width": 160,
             "height": 720,
+            "wildPokemon": {
+                "oldRod": [{
```

Comment:
I would prefer it if you put an endline before items in arrays, to make the formatting indented & consistent.

```json
[
    {
        ...
    }, {
        ...
}]
```